udata/forms/fields.py

```diff
@@ -115,7 +127,13 @@ class FileField(FieldHelper, fields.FileField):
 
 
 class URLField(FieldHelper, EmptyNone, html5.URLField):
-    pass
+    def pre_validate(self, form):
```

Comment:
Are you sure it's not already checked by `html5.URLField` ? If it's not the case, it might be better to stick with WTForms validator, ie. [`wtforms.validators.URL`](http://wtforms.readthedocs.io/en/latest/validators.html#wtforms.validators.URL)